src/redux/users/users.js

```diff
@@ -42,9 +45,12 @@ export const userLogIn = createAsyncThunk(SIGN_IN, async (obj, { dispatch }) =>
     const data = response;
     dispatch({
       type: SIGN_IN,
-      payload: data,
+      payload: {
+        ...data.data.data,
+        authenticated: true,
+      },
     });
-    localStorage.setItem('userDetails', JSON.stringify(data.data.data));
+    localStorage.setItem('userDetails', JSON.stringify({ ...data.data.data, authenticated: true }));
     // console.log('Data: ', data.data.data);
```

Comment:
- [ ] Kindly, remove this old piece of commented code and the other that is present in this file.

Reply:
Removed. Thanks